### PR TITLE
Add LUT config patch from https://github.com/ternaustralia/dawe-rlp-vocabs/pull/3

### PR DESCRIPTION
**Pull request by edmondchuc**

### CODE REVIEW

File: dawe_vocabs/settings.py [hunk 1]
```diff
@@ -92,311 +101,429 @@
...
     ),
     LUTSchema(
         "https://dev.core-api.paratoo.tern.org.au/lut-fire-growth-forms",
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-fire-growth-forms",
         "Fire growth forms codes",
         "A collection of fire growth forms and its codes.",
         "032ed6fa-4c55-4752-b625-3e6b32672444",
         "Fire growth forms",
     ),
+    # LUTSchema(
```
Comment (edmondchuc) [inline dawe_vocabs/settings.py:499]: @junrongYu can you comment on why this is commented out?

Comment (junrongYu) [reply dawe_vocabs/settings.py:499]: hi Edmond, because the api and the uuid are not used anymore

Comment (edmondchuc) [reply dawe_vocabs/settings.py:442]: If it's not used, can we just remove it entirely instead of having it commented out?

Comment (junrongYu) [reply dawe_vocabs/settings.py:442]: yeah we can, can you delete it please?

**Review by junrongYu — APPROVED** [review]

**Pull request merged**